sale_exception/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import sale
```

Comment:
License needed as per template

Reply:
IMO license in init file is useless. Yes in template, but that's not a real problem. Just my opinion

Reply:
We still should follow common practices

Reply:
updated